configs/rules.json

```diff
@@ -419,6 +419,10 @@
     "confidence_threshold": 0.95,
     "cc": []
   },
+  "performancebug": {
```

Comment:
Why did you remove `max_days_in_cache`?

Reply:
I took it out for debugging purposes to see if it would have any impact on the result 😊

> Why did you remove `max_days_in_cache`?

Reply:
> Why did you remove `max_days_in_cache`?

I took it out for debugging purposes to see if it would have any impact on the result 😊